build-aux/snap/snapcraft.yaml

```diff
@@ -167,8 +167,15 @@ parts:
       - bin/fc-cache-v7
   apparmor:
     plugin: autotools
-    build-packages: [bison, flex, gettext, g++, pkg-config]
-    source: https://launchpad.net/apparmor/3.0/3.0.7/+download/apparmor-3.0.7.tar.gz
+    build-packages: [bison, flex, gettext, g++, pkg-config, wget]
+    source: https://launchpad.net/apparmor/3.0/3.0.8/+download/apparmor-3.0.8.tar.gz
```

Comment:
Instead of patches, maybe we can use git source directly? ie. https://code.launchpad.net/~git-ubuntu-import/ubuntu/+source/apparmor/+git/apparmor/+ref/applied/ubuntu/devel the patches applied one ? Given we treat apparmor from ubuntu as authoritative. Can use branch or tag to pin to exact builds.

Reply:
Good idea - I think I had tried this locally and it ended up failing during the build or at runtime (I don't recall which now) - but I will try again and report back.

Reply:
you may need to rerun autoreconf; and thus need more build-deps.